tensorflow/core/kernels/conv_ops_test.cc

```diff
@@ -221,7 +221,14 @@ class FusedResizePadConvOpTest : public OpsTestBase {
     std::vector<Tensor> fused_tensors;
     TF_ASSERT_OK(session->Run({}, {"fused_conv"}, {}, &fused_tensors));
 
+#ifdef INTEL_MKL
```

Comment:
Please get rid of #ifdef and use ExpectClose when running with or without MKL. Checking for relative error instead of the absolute one is better in any case.